python/dgl/distributed/graph_partition_book.py

```diff
@@ -78,6 +141,18 @@ def __init__(self, part_id, num_parts, node_map, edge_map, part_graph):
         self._edge_size = len(self.partid2eids(part_id))
         self._node_size = len(self.partid2nids(part_id))
 
+    def shared_memory(self, graph_name):
+        """Move data to shared memory.
+
+        Parameters
+        ----------
+        graph_name : str
+            The graph name
+        """
+        self._meta, self._nid2partid, self._eid2partid = _move_metadata_to_shared_mam(
+            graph_name, self.num_nodes(), self.num_edges(), self._part_id, self._num_partitions,
+            self._nid2partid, self._eid2partid)
+
```

Comment:
So we will not put GraphPartitionBook to ServerState, right?

Reply:
we can have graph partition book referenced by server state. but we don't need to have it managed by server state.